prow/git/git.go

```diff
@@ -400,7 +426,7 @@ func (r *Repo) Push(branch string) error {
 	if r.user == "" || r.pass == "" {
 		return errors.New("cannot push without credentials - configure your git client")
 	}
-	r.logger.Infof("Pushing to '%s/%s (branch: %s)'.", r.user, r.repo, branch)
+	r.logger.Infof("Pushing to '%s/%s (branch: %s)'.", r.org, r.repo, branch)
```

Comment:
This doesn't seem right. Does it break any tests?

Reply:
I guess the tests assumed that the user is the same with org.